setup.py

```diff
@@ -151,8 +151,11 @@ def test_otio():
             'coverage==4.5',
         ]
     },
+
     test_suite='setup.test_otio',
 
+    tests_require=['mock;python_version<"3.3"'],
```

Comment:
The mock library became standard in Python 3.3, let me know if it's okay to introduce this dependency for testing only. If not, I can try and work out a testing strategy that omits it, but it may be pretty tricky.